app/controllers/api/v1/bookmarks_controller.rb

```diff
@@ -0,0 +1,128 @@
+class Api::V1::BookmarksController < Api::V1::BaseController
+  respond_to :json
+
+  def create
+    archivist = User.find_by_login(params[:archivist])
+    external_bookmarks = params[:bookmarks]
```

Comment:
here is where I think the naming is confusing me -- these are just bookmarks and they happen to be of external works. Can we just call them bookmarks throughout? it looks to me like this code would actually work just fine for a NON external work too.